ide/provisioning/install-debian-packages.sh

```diff
@@ -42,11 +43,12 @@ apt-get install $APTOPTS make g++ \
   librsvg2-bin xsltproc \
   imagemagick gettext \
   mesa-common-dev libgl1-mesa-dev libegl1-mesa-dev \
-  fonts-dejavu
+  fonts-dejavu \
+  libnetcdf-dev
 echo
 
 echo Installing dependencies for compiling with LLVM / Clang...
-apt-get install $APTOPTS llvm clang libc++-dev
+apt-get install $APTOPTS llvm clang-4.0 libc++-dev
```

Comment:
Why hard-code the clang version? This is fragile, and 4.0 is an ancient version.

Reply:
Sorry, yet another left-over from xcsoar-docker-build (using debian stretch). Removed.